docs/en/connector-v2/sink/Iceberg.md

```diff
@@ -76,6 +76,7 @@ libfb303-xxx.jar
 | iceberg.table.upsert-mode-enabled      | boolean | no       | false                        | Set to `true` to enable upsert mode, default is `false`                                                                                                                                                                                                                                                                   |
 | schema_save_mode                       | Enum    | no       | CREATE_SCHEMA_WHEN_NOT_EXIST | the schema save mode, please refer to `schema_save_mode` below                                                                                                                                                                                                                                                            |
 | data_save_mode                         | Enum    | no       | APPEND_DATA                  | the data save mode, please refer to `data_save_mode` below                                                                                                                                                                                                                                                                |
+| custom_sql                             | string  | no       | -                            | Custom `delete` data sql for data save mode. e.g: `delete from ... where ...`                                                                                                                                                                                                                                             |
```

Comment:
Basic lgtm. But I'm a little worried, is that the all connector of not support sql we would use our own sql style to support delete by SQL?

Reply:
No, iceberg has a data deletion API that is very similar to the SQL language style, which is the premise for implementing this function.

Reply:
provide some link into doc?

Reply:
Please look at this method

https://github.com/apache/seatunnel/pull/8094/files#diff-5e3cef996b96a634ac90ed217358f5d43fe5ccfdf05a8518a7403e5f43aafcb4R72